app/controllers/concerns/course/forum/topic/hide_concern.rb

```diff
@@ -0,0 +1,23 @@
+module Course::Forum::Topic::HideConcern
```

Comment:
Your concern is not structured properly. This is very obvious that you are trying to keep functions below the 10 line limit. Restructure.